graphql/keyspace_schema.go

```diff
@@ -302,6 +310,10 @@ func (s *KeyspaceGraphQLSchema) buildTableTypes(keyspace *gocql.KeyspaceMetadata
 			}
 		}
 
+		if err == nil && (len(inputOperatorFields) == 0 || len(inputFields) == 0 || len(fields) == 0) {
+			err = fmt.Errorf("Value, scalar, or input array empty - perhaps a table with one unfilterable column?")
```

Comment:
Error strings shouldn't be capitalized or end in punctuation. 

https://github.com/golang/go/wiki/CodeReviewComments#error-strings